dodola/services.py

```diff
@@ -126,7 +126,7 @@ def rechunk(x, target_chunks, out, max_mem, storage):
 
 
 @log_service
-def regrid(x, out, method, storage, weights_path=None, target_resolution=1.0):
+def regrid(x, out, method, storage, weights_path=None, domain_file=None):
```

Comment:
This will cause problems -- and stop me if I misunderstand what's going on here. I think the `domain_file` should not have the `None` default, as it's a required argument now.